pyproject.toml

```diff
@@ -28,9 +28,11 @@ classifiers = [
 ]
 requires-python = ">=3.9,<3.13"
 dependencies = [
+    "boltons",
```

Comment:
`OrderedMultiDict` seems to be more or less stable [since a prerelease version](https://github.com/mahmoud/boltons/blob/master/CHANGELOG.md#030) so I omitted bounds

Reply:
added a v1 lower bound just in case.